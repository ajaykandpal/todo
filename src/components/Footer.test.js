import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';
describe('Footer', () => {
  it('should render correctly with no props', () => {
    const component = shallow(<Footer/>);
    
    expect(component).toMatchSnapshot();
  });
  it('should render banner text correctly with given strings', () => {
    const strings = ['one', 'two','three'];
    const component = shallow(<Footer list={strings} />);
    expect(component).toMatchSnapshot();
  });
});

