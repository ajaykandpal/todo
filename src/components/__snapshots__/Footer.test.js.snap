// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render banner text correctly with given strings 1`] = `
<div>
  <span>
    Show: 
  </span>
  <Connect(Link)
    filter="SHOW_ALL"
  >
    All
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_ACTIVE"
  >
    Active
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_COMPLETED"
  >
    Completed
  </Connect(Link)>
</div>
`;

exports[`Footer should render correctly with no props 1`] = `
<div>
  <span>
    Show: 
  </span>
  <Connect(Link)
    filter="SHOW_ALL"
  >
    All
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_ACTIVE"
  >
    Active
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_COMPLETED"
  >
    Completed
  </Connect(Link)>
</div>
`;
